### ISSUE TITLE
[v2 Beta] Bug in Pdo.start

### ISSUE THREAD

**Issue by bdcdekimo** [Daxbot/node-canopen#17]
In pdo.js:101-102, I think you the entry in the Eds should be used instead of the dataObject stored in the Pdo:

```
                        let entry = this.device.eds.getEntry(dataObject.index);
                        if(entry.subNumber > 0)
                            entry = entry[dataObject.subIndex];

                        this.events.push([entry, 'update', listener]);
                        entry.on('update', listener);
```

It confused me at first, but pdo.dataObjects does not contain objects of class DataObject. That's why dataObject.on() does not exist.

**Comment by wilkinsw** [comment]
Do you have your PDOs mapped properly in TPDO communication parameter?  Do you have an existing EDS file you are using, or are you constructing one?

**Comment by wilkinsw** [comment]
Also have you looked at the TPDO example?
https://github.com/DaxBot/node-canopen/blob/v2-beta/examples/transmit_pdo.js

I think this might just be a configuration issue

**Comment by wilkinsw** [comment]
oh wait, no, I see what you mean, fixing now

**Comment by wilkinsw** [comment]
Give version [2.5.1](https://www.npmjs.com/package/canopen/v/2.5.1-beta) a try and see if that fixes your issue

**Comment by bdcdekimo** [comment]
Thanks for the quick fixes! 2.5.1 fixed the issue, but now that I can use Pdo.start() I discovered a few more problems. I'll open some new issues

**Comment by wilkinsw** [comment]
I appreciate you taking the time to post! I haven't done much with the PDO sending so having someone test it (and find my mistakes) is quite helpful
